overlord/ifacestate/helpers.go

```diff
@@ -677,6 +679,11 @@ func (m *IdentityMapper) RemapSnapFromRequest(snapName string) string {
 	return snapName
 }
 
+// Returns actual name of the system snap - needs to be reimplemented by concrete mapper.
+func (m *IdentityMapper) SystemSnapName() string {
```

Comment:
You don't need to provide this. If it's just for the tests, you can put it in `export_test.go`.

Reply:
Good idea, thanks! Done.